common/src/main/java/tachyon/Constants.java

```diff
@@ -35,6 +35,17 @@
   public static final String ANSI_CYAN = "\u001B[36m";
   public static final String ANSI_WHITE = "\u001B[37m";
 
+  public static final String LS_FORMAT_FILE_SIZE = "%-10s";
+  public static final String LS_FORMAT_CREATE_TIME = "%-25s";
+  public static final String LS_FORMAT_FILE_TYPE = "%-15s";
+  public static final String LS_FORMAT_PERMISSION = "%-15s";
+  public static final String LS_FORMAT_USER_NAME = "%-15s";
+  public static final String LS_FORMAT_GROUP_NAME = "%-15s";
+  public static final String LS_FORMAT_FILE_PATH = "%-5s";
+  public static final String COMMAND_FORMAT_LS = LS_FORMAT_FILE_SIZE + LS_FORMAT_CREATE_TIME
```

Comment:
It seems this change conflicts with the one in https://github.com/amplab/tachyon/pull/2406/files ?